### PR TITLE
Fix for issue #122

### PR DESCRIPTION
**Pull request by upodroid**
Bug fix

Sorry for the duplicate PR, I was unable to modify the previous PR.

### CODE REVIEW

File: src/Primrose/BrowserEnvironment.js [hunk 1]
```diff
@@ -366,16 +367,11 @@ export default class BrowserEnvironment extends EventDispatcher {
     };
 
     const doPicking = () => {
-      updateAll();
+      updateAll();  
       this.input.resolvePicking(this.scene);
     };
 
-    const moveGround = () => {
```
Comment (capnmidnight) [inline src/Primrose/BrowserEnvironment.js]: Oh! We need that line. Everything else about your PR here is right, though.

There is another issue, #123, that--if it had been addressed already--would mean that this line could be deleted. But that's not the case yet. I guess I could have broken up issue #122 and #123 to not reference each other and just waited for them to be addressed before creating a new issue for deleting this `moveGround` function.

**Comment by capnmidnight** [comment]
Hi there! I'll be writing a "getting started" document very soon that will cover how to be able to make changes on Primrose and then test those changes. I'm sorry, I didn't expect people to come on to the project so soon after I made the announcement on Twitter, so I'm not really fully prepared with documentation yet.

With the project set up completely so you can run the code after making edits to it, you'll be able to see when changes you make don't have the desired effect. In this particular case, the desired effect is "nothing changes about how the apps run." This is just a cleanup task, what is referred to as "refactoring". Refactoring is kind of like sidestepping. It's something we do that doesn't change how the program works but hopefully makes future changes easier.

In this case, a line got deleted that now makes the script not work.

But that's ok. That's why we're here. The point is not that I can do the change on my own. The point is that the change is an opportunity for you to practice and learn.

I'll get the "Getting Started" document written tomorrow. We can then get you set up so you can run the project directly and see the result of your changes. Then, we can discuss what deleting that particular line of code did, and how to tell it should have stayed.

File: src/Primrose/BrowserEnvironment.js (hunk 1 above)
Comment (upodroid) [inline src/Primrose/BrowserEnvironment.js]: O I see.

I'll readd that line and have a look at #123 .

Comment (capnmidnight) [inline src/Primrose/BrowserEnvironment.js]: Is this something you're still interested in doing? I can take care of it if you're too busy.

Comment (upodroid) [inline src/Primrose/BrowserEnvironment.js]: I'll get it fixed, just wondering how to test it

**Comment by capnmidnight** [comment]
Sorry for the delay. I didn't realize how unprepared I was to handle new people coming on board. Thank you for your patience.

For being able to test the project, you should be able to [follow these instructions](https://github.com/NotionTheory/Primrose/wiki/Working-on-Primrose).

I can see from the code that everything is right now, so I can just merge this in now.

**Pull request merged**